docs/serving/samples/hello-world/helloworld-csharp/helloworld-csharp.csproj

```diff
@@ -1,15 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2.2</TargetFramework>
+    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>

```

Comment:
Setting `AspNetCoreHostingModel` here couples the sample app to a particular hosting model. I think this should be avoided for a docs sample app.